content/03.text-mining.md

```diff
@@ -7,8 +7,50 @@
 
 ### Constructing Databases and Manual Curation
 
-1. Talk about papers that construct knowledge graphs without text mining approaches
-2. Discuss the positives and negatives for these methods
+Constructing a database is an extensive process that can date back to the early 1990s [@pmid:7584356].
+This process involves gathering relevant text such as journal articles, abstracts, or web-based text.
+Each collection of text undergoes the process of tagging and normalizing important keywords to represent entities of interest.
+This task is commonly known as named entity recognition. 
+Typically, this task uses a dictionary look-up approach [@doi:10.1093/database/bas037] or a machine learning system [@doi:10.1093/bioinformatics/btw343; @doi:10.1142/9789812776136_0062; @doi:10.1186/1471-2105-12-S8-S5] to identify the keywords of interest.
+Following named entity recognition, the next step is to detect sentences that assert a relationship of interest (i.e. relationship extraction).
+This task can be accomplished via manual curation, automated approaches or a combination of both.
+Succeeding detection, identified sentences are incorporated as evidence for establishing a relationship between mentioned entities of interest. 
```

Comment:
Take another look at this sentence. I think there's an issue with it. Are you describing after something happens or after something happens successfully?